src/main/java/org/greekleanersinc/servicetemplate/exception/TemplateNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package org.greekleanersinc.servicetemplate.exception;
+
+public class TemplateNotFoundException extends RuntimeException {
+    public TemplateNotFoundException(String message) {
```

Comment:
Medium: Can we have a generic "ResourceNotFoundException" to not create an exception class for every entity type? Also, we might want to have some shared lib to store things like this

Reply:
Nice idea, I did it.